src/main/cpp/t09_row.cpp

```diff
@@ -23,5 +23,33 @@
 using namespace std;
 
 int t09_row() {
-
+	
+	int a=1;
+	
+	int count = 1;
+	int numb = 0;
+	int countmax = 0;
+	
+	while (a != 0)
+	{
+		cin >> a;
+		if (a != numb)
+		{
+		count = 1;
+			count=1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'count' is reassigned a value before the old one has been used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)